src/app/chat/event_handling.nim

```diff
@@ -39,7 +39,7 @@ proc handleChatEvents(self: ChatController) =
     self.view.setActiveChannel(channel.chat.id)
 
   self.status.events.on("channelLeft") do(e: Args):
-    self.view.removeChat(self.view.activeChannel.chatItem.id)
+    self.view.removeChat(ChatIdArg(e).chatId)
```

Comment:
Interesting! was this causing a nil pointer exception?

Reply:
Kind of yes. When removing a channel, if the channel was not the active channel, the View/UI would remove the active one instead. So if you tried to re-add the one that was falsely removed in the UI, it would crash.
With this change, we remove the right one from the UI